widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,45 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodeMirror(Widget):
+    """Looks the same as ScriptBox but it is changed in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    ROOT = "//div[contains(@class,'miq-codemirror')]//textarea"
+    PARENT_DIV = "./.."  # we need to change visibility of this one
+    CLICK_DIV = "./../.."  # the widget has very nested divs,we need to click on that one ¯\_(ツ)_/¯
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+
+    @property
+    def element(self):
+        return self.browser.element(self)
+
+    @property
+    def is_editable(self):
+        if self.element.get_attribute("contentEditable") == "true":
+            return True
+        else:
+            return False
+
+    def make_editable(self):
```

Comment:
I'd transform this method with above property into self.editable getter/setter pattern.
you can find example here - https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py#L477

so, it would look like

```
code_mirror.editable = True
assert code_mirror.editable , "code mirror editable"
code_mirror.editable = False
assert not code_mirror.editable , "code mirror isn't editable" 
```

Reply:
When this will be merged, I can remove this completely https://github.com/ManageIQ/manageiq-ui-classic/pull/6030

I'd prefer to merge it as is and then I will do a follow-up with removing these "hacks"

Reply:
The BZ is in post https://bugzilla.redhat.com/show_bug.cgi?id=1740753

Reply:
okay, please add appropriate comment then

Reply:
@izapolsk done. There's TODO now :)